includes/admin/class-wcmp-export.php

```diff
@@ -1092,6 +1107,32 @@ public function getShipmentData(array $ids, WC_Order $order): array
         return $data;
     }
 
+    /**
+     * Returns the weight in grams.
+     * @param int $totalWeight
+     *
+     * @return float|int
+     */
+    public static function getItemWeightInGrams(int $totalWeight): float|int
```

Comment:
Oh. Dit is vies, graag alleen float laten returnen :P Hoop dat dat niet te veel effect heeft op andere plekken maar een dubbele return type is niet nice.